### DISCUSSION TITLE
Forking Distro to Build a Synth Device

### DISCUSSION THREAD

**Discussion by RK11111111111** [fayaaz/mixxx-pi-gen#80] [Ideas]
I have been building a Raspberry Pi Raspbian based synth the plan is to be like this repository is with Mixxx, just a dedicated synth device.  I've got the keyboard and synth up and running, but it is pretty far from having something that just boots to synth with a nice clean interface.  It might be interesting to just fork this build for that.   It already has alsa possibly even Jack running.

**Comment by fayaaz** [comment]
You can fork at your leisure - you must obviously adhere to the license. Enjoy!

You can install pipewire and enable it - this is where I've purposefully disabled it:
 https://github.com/fayaaz/mixxx-pi-gen/blob/main/stage3/04-enable-wayland/01-run.sh
 
 You will need to enable Github Actions on the new repo if you want it to also build your image automatically.
